public/static/docs/command-reference/get-url.md

```diff
@@ -36,14 +35,19 @@ single-purpose command that can be used out of the box after installing DVC.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Discussion     | URL format                                 |
+| Type    | Description    | `url` format                               |
 | ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/file`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data.csv`                   |
-| `gs`    | Google Storage | `gs://mybucket/data.csv`                   |
-| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data.csv` |
-| `hdfs`  | HDFS           | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file   | `https://example.com/path/to/data.csv`     |
+| `local` | Local path     | `/path/to/local/data`                      |
+| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
+| `gs`    | Google Storage | `gs://mybucket/data`                       |
+| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data`     |
+| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
+| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+
+> \* HDFS and HTTP **do not** support downloading entire directories, only
```

Comment:
did we check re the HDFS one?

Reply:
See https://github.com/iterative/dvc.org/pull/931#pullrequestreview-348279586 below. Updating...

Reply:
> only local, s3, gs, ssh support dirs 

Dah ok that's all the remaining ones. No update needed! Resolving